integration/management-api-tests/src/test/java/org/wso2/micro/integrator/api/DataSourceResourceTestCase.java

```diff
@@ -17,26 +17,27 @@
  */
 package org.wso2.micro.integrator.api;
 
-import org.apache.http.HttpResponse;
-import org.awaitility.Awaitility;
 import org.json.JSONObject;
 import org.testng.Assert;
 import org.testng.annotations.AfterClass;
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 import org.wso2.esb.integration.common.utils.ESBIntegrationTest;
-import org.wso2.esb.integration.common.utils.clients.SimpleHttpClient;
 
 import java.io.IOException;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.concurrent.TimeUnit;
+
+import static org.wso2.micro.integrator.api.Constants.COUNT;
+import static org.wso2.micro.integrator.api.Constants.LIST;
 
 public class DataSourceResourceTestCase extends ESBIntegrationTest {
 
+    private String accessToken;
+    private String endpoint;
     @BeforeClass(alwaysRun = true)
```

Comment:
Shall we add a new line before this method?

Reply:
fixed